### ISSUE TITLE
govendor equivalent for godep save

### ISSUE THREAD

**Issue by jpillora** [kardianos/govendor#80]
this works:
```
godep save && govendor migrate auto
```
though maybe there's a better way?

**Comment by kardianos** [comment]
@jpillora Could you describe what you are wanting to accomplish? What movement of files do you want to see?

**Comment by jpillora** [comment]
`godep save` looks at all external dependencies for my app and copies them locally into `Godeps/`

I know I can do `govendor add foo`, then `govendor add bar`, etc. though is there a way to just add all external deps at once?

**Comment by jpillora** [comment]
Found it `govendor add +external`

**Comment by jpillora** [comment]
I tried `govendor add` and help appeared, so I thought I was missing the path. No paths worked.

Maybe
```
govendor {add, update, remove} [-n] [-short | -long] [+status] [import-path-filter]
```
Could become
```
govendor {add, update, remove} [-n] [-short | -long] +<status> [import-path-filter]
```
Since the `[]` generally denote optional and also it will make it match `list` which has `+<status>`

**Comment by kardianos** [comment]
Fair points. Thanks for taking time to write that up. I've filed a new issue and linked it to this one.

**Comment by jpillora** [comment]
No worries, and thanks for tackling vendoring in Go - I think this tool will get more traction once 1.6 lands

**Comment by gm42** [comment]
Isn't it worth mentioning in documentation how to create an useful `vendor.json` starting from a random project X checked out via git? And I mean by automatically creating the `vendor.json` based on what's actually needed to build the binary(ies) of such project, which can then of course be adjusted manually.

I have not been able to use the `add` command to achieve this so far; I would probably mention how the `add` command works also directly in the CLI help.

For example:
```
$ govendor add
Error: missing package or status
govendor add [options] ( +status or import-path-filter )
	Copy one or more packages into the vendor folder.
	Options:
		-n           dry run and print actions that would be taken
		-tree        copy package(s) and all sub-folders under each package
		
		The following may be replaced with something else in the future.
		-short       if conflict, take short path 
		-long        if conflict, take long path
```

It's not mentioning `+external` or how that would work.
